evennia/accounts/tests.py

```diff
@@ -158,3 +161,122 @@ def test_puppet_object_already_puppeted(self):
         account.puppet_object(self.s1, obj)
         self.assertTrue(self.s1.data_out.call_args[1]['text'].endswith("is already puppeted by another Account."))
         self.assertIsNone(obj.at_post_puppet.call_args)
+
+    def test_at_look(self):
+        "Check at_look method called"
+
+        import evennia.server.sessionhandler
+
+        account = create.create_account("TestAccount%s" % randint(0, 999999), email="test@test.com", password="testpassword", typeclass=DefaultAccount)
+        self.s1.uid = account.uid
+        evennia.server.sessionhandler.SESSIONS[self.s1.uid] = self.s1
+
+        self.s1.logged_in = True
+        self.s1.protocol_key = 'dummy protocol key'
+        self.s1.address = 'dummy address'
+
+        account.at_look()
```

Comment:
What is actually tested here, apart from making sure this does not traceback? The doc string suggests this tests that at_look_method is called, but you call it yourself here, and there doesn't seem to be any asserts or analysis of the outcome?